launcher/sdw_updater_gui/Updater.py

```diff
@@ -446,19 +447,27 @@ def shutdown_and_start_vms():
 
 def _safely_shutdown_vm(vm):
     try:
-        subprocess.check_call(["qvm-shutdown", "--wait", vm])
+        subprocess.check_output(["qvm-shutdown", "--wait", vm], stderr=subprocess.PIPE)
     except subprocess.CalledProcessError as e:
         sdlog.error("Failed to shut down {}".format(vm))
         sdlog.error(str(e))
+        sdlog.error(str(e.stderr))
         return UpdateStatus.UPDATES_FAILED
 
 
 def _safely_start_vm(vm):
     try:
-        subprocess.check_call(["qvm-start", "--skip-if-running", vm])
+        running_vms = subprocess.check_output(
+            ["qvm-ls", "--running", "--raw-list"], stderr=subprocess.PIPE
+        )
+        sdlog.info("VMs running before start of {}: {}".format(vm, running_vms))
```

Comment:
This will produce logs like:

> 2020-04-06 15:12:53,712 - sdw_updater_gui.Updater:463(_safely_start_vm) INFO: VMs running before start of sd-app: b'dom0\nsd-gpg\nsd-log\nsd-proxy\nsd-whonix\nsys-firewall\nsys-net\nsys-usb\nsys-whonix\nwork\n'

I would recommend converting the list of VMs to a comma-separated list for readability.

Reply:
Done. New format looks like:

```
2020-04-06 16:13:37,826 - sdw_updater_gui.Updater:463(_safely_start_vm) INFO: VMs running before start of sd-app: ['dom0', 'fpf-infra', 'fpf-vault', 'fpf-vpn', 'fpf-web', 'sd-dev', 'sd-devices', 'sd-gpg', 'sd-kernel-builder', 'sd-log', 'sd-proxy', 'sd-whonix', 'sys-firewall', 'sys-firewall-inner', 'sys-net', 'sys-usb', 'sys-whonix']
```